### PR TITLE
✨Add favourite feature with local storage

### PR DESCRIPTION
**Pull request by Sina-Sche**

### CODE REVIEW

**Review by kschmitz123 — APPROVED** [review]
really cool ★

**Review by boldCM — APPROVED** [review]
You're styling looks awesome! 😱😍😍

File: src/components/character.css [hunk 1]
```diff
@@ -32,3 +32,26 @@
...
+
+.favourite__btn {
+  background: none;
+  cursor: pointer;
+  border: none;
+  margin-top: 10px;
+}
+.favourite__btn:hover {
+  border: 1px solid white;
+}
```
Comment (phlgr) [inline src/components/character.css:51]: On hover the button shifts the whole layout a bit.
You could use an outline so it doesn't shift the layout.
```suggestion
.favourite__btn:hover {
  outline: 1px solid white;
}
```

File: src/components/Character.js [hunk 2]
```diff
@@ -13,9 +15,50 @@ export function Character({ name, imgSrc }) {
...
+      //condition: currentFavorites already contain selected name,
+      const isFavourite = currentFavourites.includes(name);
+      //if this is true, then take the currentFavourites and filter them, by removing the name of the selected favourite
+      if (isFavourite) {
+        currentFavourites = currentFavourites.filter(
+          //!== strict inequality operator - favourite can not be equal to name
+          (favouriteBtn) => favouriteBtn !== name
+        );
+
+        //if selected name is not yet a favourite, then push name to currentFavorites
```
Comment (phlgr) [inline src/components/Character.js:41]: Great comments!

**Review by phlgr — CHANGES_REQUESTED** [review]

File: src/components/character.css (hunk 1 above)
Comment (Sina-Sche) [reply src/components/character.css]: Thanks a lot!

File: src/components/Character.js (hunk 2 above)
Comment (Sina-Sche) [reply src/components/Character.js:41]: Thanks :)

**Pull request merged**